app/layout/nav_entries.phtml

```diff
@@ -1,5 +1,7 @@
-<ul id="nav_entries">
-	<li class="item"><a class="previous_entry" href="#"><?= _i('prev') ?></a></li>
-	<li class="item"><a class="up" href="#"><?= _i('up') ?></a></li>
-	<li class="item"><a class="next_entry" href="#"><?= _i('next') ?></a></li>
-</ul>
\ No newline at end of file
+<nav>
+    <ul id="nav_entries">
```

Comment:
Please don't change the whitespace. Otherwise lgtm.

Reply:
fixed